wiki/Article_styling_criteria/en.md

```diff
@@ -11,6 +11,8 @@ The article styling criteria (ASC) serve as the osu! wiki's enforced styling sta
 
 All articles should aim at having proper grammar, correct spelling, and correct information.
 
+These articles are primarily tools to aid in reviewing, and the information presented represents the consensus of osu! wiki contributors formed over the years. Since the wiki is a collaborative effort through the review process, it is not necessary to read or memorise all of the ASC at once. If you are looking to contribute, read the [contribution guide](/wiki/osu!_wiki/Contribution_guide).
```

Comment:
I know that you are referring to the writing and formatting guidelines slash rules, but the first sentence sounds like "these articles" from its beginning and "all articles" from the previous one are the same.

also, "... information _presented represents_ ..."

Reply:
a) fair point

b) mind giving an alternative? it sounds fine to me

Reply:
_These articles are primarily tools to aid in reviewing and represent the consensus of osu! wiki contributors formed over the years_

Reply:
Pushed a change - I think it became a bit more eloquent if I do say so myself